rails_programming/rails_basics/deployment.md

```diff
@@ -143,3 +153,13 @@ This section contains helpful links to other content. It isn't required, so cons
 * [Heroku Custom Domains](https://devcenter.heroku.com/articles/custom-domains) help file
 * [Converting from PostgreSQL to SQLite3](http://manuel.manuelles.nl/blog/2012/01/18/convert-postgresql-to-sqlite/)
 * [Webinar on Using Heroku in Production](https://blog.heroku.com/archives/2013/7/11/running-production-apps-on-heroku)
+
+### Knowledge Check
+This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
+
+* <a class="knowledge-check-link" href="#dyno-knowledge-check">What are dynos? How many of them are required to run your application?</a>
```

Comment:
I gave it some thought and I think a question of "Do you need more dynos for more web traffic? would be a bit better. What are dynos is a bit too broad and I think the core essence of that paragraph is essential that you need more instances/dynos for more traffic. So if we could change it to that I would be okay with the first knowledge check. You can also just drop it if you want since learners probably aren't going to have to worry about getting a ton of traffic for their Odin projects if you want as well. It's more a nice to know while doing TOP.